query-service-impl/src/test/resources/prometheus_request_handler.conf

```diff
@@ -0,0 +1,36 @@
+{
+  name = raw-service-view-service-prometheus-handler
+  type = prometheus
+  clientConfig = ""
+  requestHandlerInfo {
+    tenantAttributeName = tenant_id
+    startTimeAttributeName = "SERVICE.startTime"
+    prometheusViewDefinition {
+      viewName = rawServiceView
+      metricScope = SERVICE
+      metricMap {
+        numCalls {
+          metricName: "num_calls",
+          metricType: "GAUGE"
+        },
+        errorCount {
+          metricName: "error_count",
+          metricType: "GAUGE"
+        }
+      }
+      attributeMap {
+        "SERVICE.id": "service_id",
+        "SERVICE.name": "service_name",
+        "API.id": "api_id",
+        "API.name": "api_name",
+        "EVENT.protocolName": "protocol_name",
+        "EVENT.status_code": "status_code",
+        "API_TRACE.status_code": "status_code",
+        "API.startTime": "start_time_millis",
```

Comment:
Can we remove this `"API.startTime": "start_time_millis",
        "API.endTime": "end_time_millis",`?